panel/models/tabulator.ts

```diff
@@ -816,33 +822,23 @@ export class DataTabulatorView extends HTMLBoxView {
     return lookup
   }
 
-  renderChildren(all: boolean = true): void {
-    new Promise(async (resolve: any) => {
-      let new_children = await this.build_child_views()
-      if (all) {
-        new_children = this.child_views
-      }
-      resolve(new_children)
-    }).then((new_children) => {
+  renderChildren(): void {
+    new Promise(async () => {
```

Comment:
```suggestion
    void new Promise(async () => {
```

Nit, Raises an eslint error.